An error occurred while executing the following cell:
------------------
!git clone https://github.com/probml/pyprobml /pyprobml &> /dev/null
%cd -q /pyprobml/scripts
import pyprobml_utils as pml
import numpy as np
import matplotlib.pyplot as plt
------------------

---------------------------------------------------------------------------
ModuleNotFoundError                       Traceback (most recent call last)
Input In [1], in <cell line: 3>()
      1 get_ipython().system('git clone https://github.com/probml/pyprobml /pyprobml &> /dev/null')
      2 get_ipython().run_line_magic('cd', '-q /pyprobml/scripts')
----> 3 import pyprobml_utils as pml
      4 import numpy as np
      5 import matplotlib.pyplot as plt

ModuleNotFoundError: No module named 'pyprobml_utils'
ModuleNotFoundError: No module named 'pyprobml_utils'
